include/odp/api/spec/schedule_types.h

```diff
@@ -160,6 +160,39 @@ extern "C" {
  */
 typedef int odp_schedule_prio_t;
 
+/**
+ * Multi event config parameters
+ */
+typedef struct odp_multi_event_param_t {
+	/** Number of events to accumulate
+	 *
+	 * The scheduler will asynchronously accumulate upto these many number
+	 * of events and deliver the events on the subsequent invocation of
+	 * odp_schedule_multi().
+	 * This value should be less than or equal to max_multi_event_count.
+	 *
+	 * 0:     Asynchronous accumulation is disabled.
+	 * >= 1:  Asynchronous accumulation is enabled.
+	 *
+	 * @see odp_schedule_capability_t::max_multi_event_count.
+	 *
+	 * Default value is 0. */
+	uint32_t   multi_event_count;
+
+	/** Timeout for accumulation
+	 *
+	 * The scheduler will asynchronously accumulate events for this time
+	 * period before delivering the batch of events.
+	 * This value should be in the range defined by
+	 * multi_event_min_async_wait and multi_event_max_async_wait.
+	 *
+	 * @see odp_schedule_capability_t::multi_event_min_async_wait.
+	 * @see odp_schedule_capability_t::multi_event_max_async_wait.
+	 *
+	 * Default value is 0. */
+	uint32_t  multi_event_async_wait_ns;
+} odp_multi_event_param_t;
```

Comment:
This specification does not bring much new functionality or guarantees to the application. Biggest difference would be to an implementation which receives expected burst size (and timeout) in scheduler configuration phase. The same could be achieved with implementation specific configuration file. E.g. application would still need to prepare to receive 1...N events from each schedule multi call.

An schedule burst size API change should probably specify more functionality / limitations e.g.:
 * promise that exactly N events is returned if there was no timeout
 * specify that application must call odp_schedule_multi(), with 'num' >= N. odp_schedule() could not be used when N > 1.
 * how this wait time relates to schedule call wait time. Is this max (or min) schedule wait time that application will set into a schedule call.